core/templates/pages/preferences-page/preferences-page.component.html

```diff
@@ -42,7 +42,7 @@
       <div class="form-group row">
         <label for="label-target-user-bio" class="col-lg-4 col-md-4 col-sm-4" translate="I18N_PREFERENCES_BIO"></label>
         <div class="col-lg-8 col-md-8 col-sm-8">
-          <textarea id="label-target-user-bio" class="oppia-bio-border protractor-test-user-bio" ng-model="$ctrl.userBio" ng-blur="$ctrl.saveUserBio($ctrl.userBio)" rows="5" maxlength="2000"></textarea>
+          <textarea id="label-target-user-bio" class="oppia-bio-border protractor-test-user-bio" ng-model="$ctrl.userBio" ng-model-options="{ debounce: 1000 }" ng-change="$ctrl.saveUserBio($ctrl.userBio)" rows="5" maxlength="2000"></textarea>
```

Comment:
What's this issue with `ng-blur`, any specific reason for replacing it with `ng-change`? (Also, update the same in the PR description. :))

Reply:
ng-blur: saves when input box loses focus
ng-change : saves after 1second from last keystroke